### ISSUE TITLE
Flake8 fails: `plugin code for 'flake8-classmethod-staticmethod[CLST1]' does not match ^[A-Z]{1,3}[0-9]{0,3}$`

### ISSUE THREAD

**Issue by simon-liebehenschel** [atollk/flake8-classmethod-staticmethod#1]
## Log
```
flake8...................................................................Failed
- hook id: flake8
- exit code: 1

There was a critical error during execution of Flake8:
plugin code for `flake8-classmethod-staticmethod[CLST1]` does not match ^[A-Z]{1,3}[0-9]{0,3}$
```
## Config
```yaml

  - repo: https://github.com/pycqa/flake8
    rev: 5.0.4
    hooks:
      - id: flake8
        additional_dependencies:
          [
            "flake8-todos",
            "flake8-print",
            "flake8-bugbear",
            "flake8-simplify",
            "flake8-comprehensions",
            "flake8-pytest-style",
            "flake8-return",
            "flake8-classmethod-staticmethod",
            "pep8-naming",
            "wemake-python-styleguide",
          ]
```
## Versions
- Python 3.10.8
- pre-commit 2.20.0

**Comment by simon-liebehenschel** [comment]
Perhaps related: https://github.com/PyCQA/flake8/issues/1643

As I understand, the only solution to be compatible with Flake8 is migration to 3 letters codes.

**Comment by atollk** [comment]
Ugh, what an annoying change on flake8's side. Thanks for reporting. I'll see that I'll fix it sometime soon.

**Comment by wwuck** [comment]
@atollk Any update to this?

**Comment by atollk** [comment]
I totally forgot about this, my bad. I'll put it on top of my to-do list when I'm back from vacation. Sorry for the inconvenience.

**Comment by atollk** [comment]
Should be fixed in v2.0. Let me know if there are still issues.
